packages/dialog/src/action.tsx

```diff
@@ -0,0 +1,34 @@
+import {MaterialComponent} from '@preact-material-components/base/lib/MaterialComponent';
+import {h} from 'preact';
+
+export interface IDialogButtonProps {
+  scrollable?: boolean;
+}
+
+export interface IDialogButtonState {}
+
+export class DialogButton extends MaterialComponent<
+  IDialogButtonProps,
+  IDialogButtonState
+> {
+  protected componentName = 'dialog__button';
+  protected mdcProps = [];
+
+  protected materialDom(allProps) {
+    const {accept, cancel, children, props} = allProps;
+    let actionString = 'false';
+    if (accept) {
+      actionString = 'accept';
+    } else if (cancel) {
+      actionString = 'close';
```

Comment:
Just to be sure, this are the only options right?

Reply:
They can be anything but these are the default ones

Reply:
Shouldn't the user have a choice then?

Reply:
They have a choice by adding a prop, but I guess we should create a`action` prep to make it easier?

Reply:
Yes and then just replace `false` with
```typescript
action ? action : 'false' 
```

Reply:
(and add it to the docs)